docscan/session/create/filter/document_restrictions.go

```diff
@@ -0,0 +1,49 @@
+package filter
+
+// RequestedDocumentRestriction represents a document Restriction for checks and tasks
+type RequestedDocumentRestriction struct {
+	DocumentTypes []string `json:"document_types"`
+	CountryCodes  []string `json:"country_codes"`
+}
+
+// RequestedDocumentRestrictionBuilder builds a RequestedDocumentRestriction
+type RequestedDocumentRestrictionBuilder struct {
+	documentTypes []string
+	countryCodes  []string
+}
+
+// NewRequestedDocumentRestrictionBuilder creates a new RequestedDocumentRestrictionBuilder
+func NewRequestedDocumentRestrictionBuilder() *RequestedDocumentRestrictionBuilder {
+	return &RequestedDocumentRestrictionBuilder{
+		documentTypes: []string{},
+		countryCodes:  []string{},
+	}
+}
+
+// WithCountryCode adds a country code to the Restriction
+func (b *RequestedDocumentRestrictionBuilder) WithCountryCode(countryCode string) *RequestedDocumentRestrictionBuilder {
+	b.countryCodes = append(b.countryCodes, countryCode)
+	return b
+}
+
+// TODO: need country code? Looks like Python just has plural
```

Comment:
We can remove the singular methods (other SDKs don't have this)

[Java has `withCountries` and `withDocumentTypes`](https://github.com/getyoti/yoti-java-sdk/blob/master/yoti-sdk-impl/src/main/java/com/yoti/api/client/docs/session/create/filters/SimpleDocumentRestrictionBuilder.java)